portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,226 @@
-#content {
+body {
+  background-image: url('images/floral_bg.jpg');
+}
+
+.content {
+  margin-top: auto;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  padding: 20px;
+  max-width: 1000px;
+  background-color: white;
+  font-size: 20px;
+}
+
+h3 {
+  text-shadow: 1px 1px 3px #ccc;
+}
+
+.info-bar {
+  border-right: 1px solid #ccc;
+}
+
+#name {
+  font-weight: bold;
+  margin: 15px 0px;
 }
 
-#greeting-container {
+#location i {
+  font-size: 24px;
+}
+#location span {
+  font-size: 18px;
+  margin: 20px 0px;
+}
+
+#links {
   margin-top: 20px;
 }
+#links a {
+  display: block;
+  margin-top: 10px;
+  color: #7c7b8a;
+}
+#links a i {
+  font-size: 30px;
+}
+
+.fa-linkedin-square {
+  color: #308ECA;
+}
+
+.fa-github {
+  color: black;
+}
+
+.fa-envelope-o {
+  margin-top: 20px;
+  color: #c44a41;
+}
+
+/* Avatar top image, hidden by default. */
+.img-top {
+  display: none;
+}
+#avatar img {
+  max-height: 120px;
+}
+/* On hover, only displays top image. */
+#avatar:hover img {
+  display: none;
+}
+#avatar:hover .img-top {
+  display: inline;
+}
+
+.top-nav {
+  padding: 4px;
+}
+
+/* index.html */
+
+.fa-hand-o-right {
+  margin-left: 10px;
+}
+
+#my-name {
+  font-weight: 800;
+  font-size: 30px;
+  color: #3aa7d6;
+  text-shadow: 1px 2px 4px #ccc;
+}
+#my-name:hover {
+  color: #a7c1fa;
+}
+
+.text-bold {
+  font-weight: 700;
+  font-size: 22px;
+  color: #607bd1;
+}
+
+.text-red {
+  color: #b34949;
+}
+
+#talents-title:hover {
+  cursor: pointer;
+  color: #7c7b8a;
+}
+
+#fun-title {
+  margin-top: 50px;
+}
+
+#fun-div {
+  text-align: center;
+}
+
+#fun-div button {
+  border-radius: 10px;
+  color: white;
+  width: 100%;
+  padding: 8px;
+  margin-bottom: 10px;
+  background-color: #ed7979;
+  border: 2px solid #ed7979;
+  box-shadow: 5px 5px 8px #ab9da8;
+}
+#fun-div button:hover {
+  background-color: white;
+  color: black;
+}
+
+.img-fact {
+  max-width: 80px;
+}
+
+/* projects.html */
+
```

Comment:
Remove new line?

Reply:
This is a section comment and should have a new line according to the style guide.

Reply:
Ah, got it.